### ISSUE TITLE
[CI] Tests are silently failing

### ISSUE THREAD

**Issue by epicfaace** [codalab/codalab-worksheets#1322]
Make sure that if tests fail, then build fails

**Comment by epicfaace** [comment]
This happens on the master branch too. For example: https://travis-ci.org/codalab/codalab-worksheets/jobs/574502782

you get the output

```
process: [!][!] Tests failed: worksheet, unicode
process: 
The command "./codalab_service.py start -s default test --mysql-port 3306 --version ${TRAVIS_BRANCH}" exited with 0.
```

@percyliang

**Comment by percyliang** [comment]
Fixed by #1325
